src/feedback/components/AmountPicker.vue

```diff
@@ -47,25 +47,33 @@
         <QSlider
           :value="limitedValue"
           :min="0"
-          :max="70"
+          :max="100"
```

Comment:
The `100` is repeated in 3 places now, so would justify being put into a constant somewhere (or a value in the vue `data`, given we can't use a variable directly in the vue template...).